src/Modules/StatCan.OrchardCore.OpenAPI/Views/Widget-OpenAPI.liquid

```diff
@@ -0,0 +1,26 @@
+{% style name:"swagger-style", src:"~/StatCan.OrchardCore.OpenAPI/Styles/swagger-ui.css", debug_src:"~/StatCan.OrchardCore.OpenAPI/Styles/swagger-ui.css" %}
+{% script name:"swagger", src:"~/StatCan.OrchardCore.OpenAPI/Scripts/swagger-ui-bundle.js", debug_src:"~/StatCan.OrchardCore.OpenAPI/Scripts/swagger-ui-bundle.js", at:"Foot" %}
+{% script name:"swagger-preset", src:"~/StatCan.OrchardCore.OpenAPI/Scripts/swagger-ui-standalone-preset.js", debug_src:"~/StatCan.OrchardCore.OpenAPI/Scripts/swagger-ui-standalone-preset.js", at:"Foot" %}
+
+<div id="swagger-ui"></div>
```

Comment:
I would suggest to change this to a class and move the init code in a js file that looks for the classes after page load. This way you will be safe to include many on a single page without the risk of ID collisions

Reply:
Hmm... I understand. But it's also an unlikely use case. Can this enhancement be done later?

Reply:
I would at least change the id to a class